app/js/controllers/foyer/individuForm.js

```diff
@@ -7,7 +7,7 @@ angular.module('ddsApp').controller('FoyerIndividuFormCtrl', function($scope, in
     $scope.maxAgeYears = 130;
     $scope.minBirthDate = moment().subtract($scope.maxAgeYears, 'years');
 
-    if (individuRole == 'enfant') {
+    if (individuRole === 'enfant') {
```

Comment:
Mmh… en l'occurrence, cela mène à un commit unique qui prétend faire _Rely on strict equality in individuForm.js_ alors que sa raison d'être est de corriger la non-restauration d'un statut marital, qui est lié à un déplacement d'un morceau de code. Attention à l'atomicité des commits  :wink:

Pour ce qui est des choix de style, il est important de faire un choix et de s'y tenir, avec du linting le cas échéant. À défaut, chaque changement est l'occasion de modifier le code selon ses préférences personnelles. Je préfèrerais donc que ce changement ne soit pas fait dans cette PR mais fasse l'objet, si c'est important pour toi, d'une PR à part entière, qui systématise le changement et qui ajoute le lint correspondant. Il me semblerait pertinent qu'on en discute avant.